### PR TITLE
New language: PowerShell

### PR DESCRIPTION
**Pull request by Laoujin**
I started from the Bash tutorial and converted it to PS1 + Added PowerShell specific stuff.

Related Issues:
https://github.com/adambard/learnxinyminutes-docs/issues/1020
https://github.com/adambard/learnxinyminutes-docs/issues/1750

### CODE REVIEW

**Comment by Spurlow** [comment]
Thanks!

**Comment by ojosdegris** [comment]
Right now all formatting went nuts

**Comment by Laoujin** [comment]
This pull request should fix it (I hope:)
https://github.com/adambard/learnxinyminutes-docs/pull/2042

**Comment by Laoujin** [comment]
Either the website has not yet been republished or it still isn't working.
I'll look into generating the website offline and fix the issue(s)

**Comment by Laoujin** [comment]
https://github.com/adambard/learnxinyminutes-docs/pull/2051

Was missing some newlines I think.
Tried building the site myself with middleman first but didn't manage to set it up...

File: powershell.html.markdown [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+# All .NET methods and properties are thus available
+'string'.ToUpper().Replace('G', 'ggg')
+# Or more powershellish
+'string'.ToUpper() -replace 'G', 'ggg'
+
+# Unsure how that .NET method is called again?
+'string' | gm
+
+# Syntax for calling static .NET methods
+[System.Reflection.Assembly]::LoadWithPartialName('Microsoft.VisualBasic')
```
Comment (Laoujin) [inline powershell.html.markdown]: The line above is not rendered in the site

**Pull request merged**